src/Pages/AuditLogs.tsx

```diff
@@ -0,0 +1,66 @@
+import {useQuery} from '@tanstack/react-query';
+import axiosFactory from 'Helpers/axios';
+import useConfig from 'Hooks/useConfig';
+import {AuthContext} from 'Providers/AuthProvider';
+import {useContext} from 'react';
+
+type AuditLog = {
+  id: string;
+  action: string;
+  actedOn: string;
+  actionKey: string;
+  before: {[key: string]: any};
+};
+
+type ApiAuditLog = {
+  [key: string]: any;
+};
+
+const AuditLogsPage = () => {
+  const {authApiBaseUrl} = useConfig().config;
+  const {accessToken} = useContext(AuthContext).authData;
+
+  const getAuditLogs = async (baseUrl: string): Promise<AuditLog[]> => {
+    const client = axiosFactory(`https://sl-auth-test-v3-lambda.sfrefarch.com/`);
```

Comment:
Remove hard coded value. These should be pulled from the config file.

Reply:
Sorry. I have already pulled the value from config file. I was just testing for performance purposes and forgot to replace that with baseUrl.